lib/config/fileconf.go

```diff
@@ -196,6 +211,15 @@ func MakeSampleFileConfig(flags SampleFlags) (fc *FileConfig, err error) {
 		p.PublicAddr = apiutils.Strings{net.JoinHostPort(flags.ClusterName, fmt.Sprintf("%d", teleport.StandardHTTPSPort))}
 		p.WebAddr = net.JoinHostPort(defaults.BindIP, fmt.Sprintf("%d", teleport.StandardHTTPSPort))
 	}
+	if len(flags.PublicAddr) > 0 {
+		p.PublicAddr = apiutils.Strings(flags.PublicAddr)
+	}
+	if flags.KeyFile != "" && flags.CertFile != "" {
```

Comment:
Do we validate somewhere that these files actually exist, can be opened and that they form a valid keypair?

Reply:
It's validated in `teleport` binary through `ExistingFileVar` flags
```
    dump.Flag("cert-file", "Path to a TLS certificate file").ExistingFileVar(&dumpFlags.CertFile)                                
    dump.Flag("key-file", "Path to a TLS key file").ExistingFileVar(&dumpFlags.KeyFile) 
```

Reply:
@greedy52 Cool. Let's also make sure that they are a valid key pair (e.g. using `tls.LoadX509KeyPair`).